feat(sveltekit): Document new `sentryHandle` options (#8768)
getsentry/sentry-docs: src/platforms/javascript/guides/sveltekit/manual-setup.mdx

  //... your load code
});
```

### Configure Client-side `fetch` Instrumentation

<Note>

Available since version 7.91.0

</Note>

The `sentryHandle` function you added to your `handle` hook in `hooks.server.ts` during [server-side setup](#server-side-setup) injects a small inline `<script>` tag into the HTML response of the server.
This script attempts to proxy all client-side `fetch` calls, so that `fetch` calls inside your `load` functions are captured by the SDK.
However, if you configured CSP rules to block inline fetch scripts by default, this script will be [blocked by the browser](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_inline_script).
To enable the script, you need to add an exception for the `sentryHandle` script:

First, specify a nonce in the `fetchProxyScriptNonce` option in your `sentryHandle` call:

```javascript {filename:hooks.server.ts}
// Add the nonce to the <script> tag:

> export const handle = sentryHandle({ fetchProxyScriptNonce: "2726c7f26c" });

Reviewer: Is this the actual nonce users need to define or do they potentially have a different one?

Author: Ahh good point, this might be missleading. No, it can be any nonce. Just needs to match the one specified in the CSP. Probably better to replace with `fetchProxyScriptNonce: "_YOUR_NONCE_ `

```

Then, adjust your [SvelteKit CSP configuration](https://kit.svelte.dev/docs/configuration#csp):

```javascript {svelte.config.js}
const config = {
  kit: {
    csp: {
      directives: {
        "script-src": ["nonce-<your-nonce>"],
      },
    },
  },
};
```

> #### Disable Client-side `fetch` proxy script

Reviewer 2: ```suggestion
#### Disable Client-side `fetch` Proxy Script
```

> Alternatively to the above, or if you generally want to disable adding this script, you can disable it by passing `injectFetchProxyScript: false` to `sentryHandle`:

Reviewer: ```suggestion
If you do not want to inject the script responsible for instrumenting client-side `load` calls, you can disable injection by passing `injectFetchProxyScript: false` to `sentryHandle`:
```

```javascript {filename:hooks.server.ts}
export const handle = sentryHandle({ injectFetchProxyScript: false });
```

Note that if you disable the `fetch` proxy script, the SDK will not be able to capture spans for `fetch` calls made in your `load` functions on the client.

> This also means that potential spans created on the server for these `fetch` will not be connected to the client-side trace.

Reviewer 2: ```suggestion
This also means that potential spans created on the server for these `fetch` calls will not be connected to the client-side trace.
```
